backend/inspirehep/files/proxies.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from flask import current_app
+from werkzeug.local import LocalProxy
+
+s3_instance = LocalProxy(lambda: current_app.extensions["inspirehep-s3"].s3_instance)
```

Comment:
```suggestion
current_s3_instance = LocalProxy(lambda: current_app.extensions["inspirehep-s3"].s3_instance)
```
to keep the Flask naming convention?